### ISSUE TITLE
FullCalendar: django.core.urlresolvers.NoReverseMatch: Reverse for 'Scheduler_sessionoccurrencemodifier_change'

### ISSUE THREAD

**Issue by jdqx** [llazzaro/django-scheduler#222]
`django.core.urlresolvers.NoReverseMatch: Reverse for 'Scheduler_sessionoccurrencemodifier_change' with arguments '(12345,)' and keyword arguments '{}' not found. 0 pattern(s) tried: []`

Occurs at lines 16 and 24 of schedule/templates/fullcalendar_script.html

```
`function getEventViewURL(event){
    if (event.existed){
        if("{{ request.get_full_path }}".indexOf("/admin") != 0){
            var view_url = "{% url 'occurrence' 12345 6789 %}".replace(/12345/, event.event_id).replace(/6789/, event.id);
        }
        else{
            var view_url = "{% url 'admin:Scheduler_sessionoccurrencemodifier_change' 12345 %}".replace(/12345/, event.id)
        }
    }
    else {
        if("{{ request.get_full_path }}".indexOf("/admin") != 0){
            var view_url = "{% url 'event' 12345 %}".replace(/12345/, event.event_id);
        }
        else{
            var view_url = "{% url 'admin:Scheduler_sessionscheduler_change' 12345 %}".replace(/12345/, event.event_id)
        }
    }
    return view_url;
}`
```

Looks like missing admin code.

@nwaxiomatic

**Comment by jdqx** [comment]
This issue mysteriously went away, for now. Will report if it reappears again and will explain the cause. Anyone else reproduce this with fullcalendar.io templates?

**Comment by nwaxiomatic** [comment]
Hey, so I have another helper app that I use in conjunction with this one to add more properties to the events, and this is some code I had for that that I didn't see and modify. The actual code should be 

```
function getEventViewURL(event){
    if (event.existed){
        if("{{ request.get_full_path }}".indexOf("/admin") != 0){
            var view_url = "{% url 'occurrence' 12345 6789 %}".replace(/12345/, event.event_id).replace(/6789/, event.id);
        }
        else{
            var view_url = "{% url 'admin:schedule_occurrence_change' 12345 %}".replace(/12345/, event.id)
        }
    }
    else {
        if("{{ request.get_full_path }}".indexOf("/admin") != 0){
            var view_url = "{% url 'event' 12345 %}".replace(/12345/, event.event_id);
        }
        else{
            var view_url = "{% url 'admin:schedule_event_change' 12345 %}".replace(/12345/, event.event_id)
        }
    }
    return view_url;
}
```

I believe, let me know if that works.

**Comment by bcjarrett** [comment]
I had this issue earlier today, and your updated `{% url %}` paths fixed it for me

**Comment by oriregev** [comment]
I tried the fix suggested by @nwaxiomatic and I'm still getting the error.
I'm running on Django 1.9 in virtualenv

```
NoReverseMatch at /schedule/fullcalendar/testcalendar1/

Reverse for 'schedule_occurrence_change' with arguments '(12345,)' and keyword arguments '{}' not found. 0 pattern(s) tried: []

Request Method: 	GET
Request URL: 	http://localhost:8000/schedule/fullcalendar/testcalendar1/
Django Version: 	1.9
Exception Type: 	NoReverseMatch
Exception Value: 	

Reverse for 'schedule_occurrence_change' with arguments '(12345,)' and keyword arguments '{}' not found. 0 pattern(s) tried: []

Error during template rendering

In template /Users/ori/.virtualenvs/vl-django1.9/lib/python2.7/site-packages/schedule/templates/fullcalendar.html, error at line 0
Reverse for 'schedule_occurrence_change' with arguments '(12345,)' and keyword arguments '{}' not found. 0 pattern(s) tried: []
1 	{% extends 'site_base.html' %}
2 	{% load i18n staticfiles scheduletags %}
3 	{% load static from staticfiles %}
4 	
5 	{% block head_title %}Calendar: {{ object.name }}{% endblock %}
6 	{% block tab_id %}id='home_tab'{% endblock %}
7 	{% block extra_head %}
8 		<link rel="stylesheet" href="{% static 'schedule/css/modal.css' %}" />
9 	    {% include "fullcalendar_script.html" %}
10 	{% endblock %}
```

**Comment by Echel8n** [comment]
I have the same problem with django 1.9 and 1.10.
What information can I git to you for solving the bug ?

**Comment by mpaolino** [comment]
Same here, Django 1.10, django-scheduler 0.8.2

**Comment by llazzaro** [comment]
I hope to have a fix for 0.8.3 soon. I will give priority to this issue.

**Comment by friendka** [comment]
@oriregev you got such error as Occurrence model is not registered in admin section
